docs/tasks/administer-cluster/my-scheduler.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```

Comment:
we should instead avoid adding this comment considering that even earlier versions will need apps/v1beta1 or extensions group. In the master branch, we can safely assume users are reading docs about v1.9.0+. Docs for older supported versions can be found elsewhere.

Reply:
thanks for your review,you mean that delete the comment and only modify to apps/v1?

Reply:
yes